pkg/scheduler/framework/v1alpha1/waiting_pods_map.go

```diff
@@ -100,6 +135,10 @@ func (w *waitingPod) Allow() bool {
 // Reject declares the waiting pod unschedulable. Returns true if the allow signal
```

Comment:
Could you fix the comment too? It says allow signal when it means reject signal

Reply:
Fixed.